proposed/longhorn/latest/questions.yml

```diff
@@ -2,6 +2,7 @@ categories:
 - storage
 labels:
   io.rancher.certified: experimental
+namespace: longhorn-system
 questions:
 - variable: defaultImage
```

Comment:
@yasker IMO we should completely remove this question and the image tag sub-questions. The deployed image versions are not something to be messed with by a regular user; if a developer wants this, they can use their own.

Also note that upgrading to a newer catalog with different defaults for these questions will not change the question answers automatically. Consequently, upgrading from `0.3.0` to theoretical release `0.3.1` will never update the image tags. The upgrade will seem to succeed but no images change.

Reply:
Yeah, if the old value was used as the default engine in the new version of manager, that's really dangerous.

If we can find a way to use the default value (instead of user-specified value) every time when the user selected app, the question will still be useful. Otherwise, let's get rid of this option and hard code it to the chart.